src/context/towerContext.js

```diff
@@ -192,6 +192,26 @@ class Towers {
             }
         });
     }
+
+    /**
+     * Create a new channel given a name
+     */
+    * createChannel(towerId, newChannelName) {
+        yield new ChannelsApi(this.cityConfig(towerId)).createChannel({
+            towerId: towerId,
+            channel: {name: newChannelName}
+        }).then(channel => {
```

Comment:
If you're not doing anything with the result here, you don't need the `.then()` and can go straight to the catch clause.